.github/workflows/close-inactive-issues.yml

```diff
@@ -0,0 +1,24 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v3
+        with:
+          any-of-labels: "Status: Waiting for Customer"
```

Comment:
I think we might have to add the emoji also here because, from what I can tell after taking a look at [the action code](https://github.com/actions/stale/blob/57708ae6dd2006b5642c4e1e88bac7efb3d9382e/src/classes/issues-processor.ts#L353), it will search for a label which have the same exact name

Reply:
@mmaltsev-ct did you find this change is not needed?

Reply:
@CarlosCortizasCT no, I think I just somehow missed your comment 🤷‍♂️. Nice catch, thank you!